internal/interfaces/svm.go

```diff
@@ -24,27 +24,29 @@ type SvmDataModelONTAP struct {
 
 // SvmResourceModel describes the resource data model.
 type SvmResourceModel struct {
-	Name           string              `mapstructure:"name,omitempty"`
-	Ipspace        Ipspace             `mapstructure:"ipspace"`
-	SnapshotPolicy SnapshotPolicy      `mapstructure:"snapshot_policy,omitempty"`
-	SubType        string              `mapstructure:"subtype,omitempty"`
+	Aggregates     []map[string]string `mapstructure:"aggregates"`
 	Comment        string              `mapstructure:"comment"`
+	Ipspace        Ipspace             `mapstructure:"ipspace"`
 	Language       string              `mapstructure:"language,omitempty"`
 	MaxVolumes     string              `mapstructure:"max_volumes,omitempty"`
-	Aggregates     []map[string]string `mapstructure:"aggregates"`
+	Name           string              `mapstructure:"name,omitempty"`
+	SnapshotPolicy SnapshotPolicy      `mapstructure:"snapshot_policy,omitempty"`
+	Storage        Storage             `mapstructure:"storage"`
```

Comment:
Since it's optional, it should be set omitempty.

Reply:
You may check https://github.com/NetApp/terraform-provider-netapp-ontap/blob/integration/main/internal/provider/storage/storage_volume_resource.go#L647

Reply:
Understood. I have removed the explicit `setStorageLimitEmpty` parameter, and instead adapted the same logic as in [StorageVolumeResource.PercentSnapshotSpace](https://github.com/NetApp/terraform-provider-netapp-ontap/blob/integration/main/internal/provider/storage/storage_volume_resource.go#L647).
Please let me know if this is acceptable.